packages/uui-ref-node/lib/uui-ref-node.element.ts

```diff
@@ -60,11 +79,37 @@ export class UUIRefNodeElement extends UUIRefElement {
     ></small>`;
   }
 
-  private _renderFallbackIcon() {
+  #renderFallbackIcon() {
     return html`<uui-icon .svg="${this.fallbackIcon}"></uui-icon>`;
   }
 
-  public render() {
+  #renderContent() {
+    return html`
+      <span id="icon">
+        <slot name="icon" @slotchange=${this.#onSlotIconChange}></slot>
+        ${this._iconSlotHasContent === false ? this.#renderFallbackIcon() : ''}
+      </span>
+      <div id="info">
+        <div id="name">${this.name}</div>
+        ${this.renderDetail()}
+      </div>
+    `;
+  }
+
+  #renderLink() {
+    return html`<a
+      id="open-part"
+      tabindex=${this.disabled ? (nothing as any) : '0'}
+      href=${ifDefined(!this.disabled ? this.href : undefined)}
+      target=${ifDefined(this.target || undefined)}
+      rel=${ifDefined(
+        this.target === '_blank' ? 'noopener noreferrer' : undefined
```

Comment:
Not entirely sure about this; for one, setting `noreferrer` should be enough as that implies `noopener` ([MDN source](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer)), secondly, it seems a stretch that all _blank links would require a noreferrer as they could be internal links and therefore you would lose tracking if you kept this on.

Consider adding this as a real property on the element instead, as there are a host of other values you can set in this field: [rel-attribute on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel)

Reply:
I think this is just how we have done it on all the other Components that supports HREF.

So I think this should be changed as one more general PR.

I'm not sure how relevant it is now, but could we create it as a task on AzureBoards?

Reply:
@nielslyngsoe deal